### PR TITLE
Add display to WA templates button components

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/utils/whatsapp/tests.py [hunk 1]
```diff
@@ -275,11 +275,15 @@ def test_update_local_templates_whatsapp(self):
...
                 },
                 "footer": {"content": "Thanks for your patience", "params": []},
-                "button.0": {"content": "Yes", "params": []},
-                "button.1": {"content": "No", "params": []},
-                "button.2": {"content": "Call center", "params": []},
-                "button.3": {"content": "Check website", "params": [{"type": "text"}]},
-                "button.4": {"content": "Check website", "params": []},
+                "button.0": {"content": "Yes", "display": "Yes", "params": []},
+                "button.1": {"content": "No", "display": "No", "params": []},
+                "button.2": {"content": "Call center", "display": "Call center", "params": []},
```
Comment (rowanseymour) [inline temba/utils/whatsapp/tests.py:280]: I don't think we need this duplication.. just omit display where there's only content

File: temba/utils/whatsapp/tasks.py [hunk 2]
```diff
@@ -132,13 +135,15 @@ def _extract_template_params(components):
             buttons = component["buttons"]
             for idx, button in enumerate(buttons):
                 comp_params = []
+                content = button.get("text", "")
+                display = button.get("text", "")
                 if button["type"].lower() == "url":
                     for match in VARIABLE_RE.findall(button.get("url", "")):
                         comp_params.append({"type": "text"})
-                        all_parts_supported = False
+                    content = button.get("url", "")
```
Comment (rowanseymour) [inline temba/utils/whatsapp/tasks.py:143]: do we need courier changes to make buttons with urls work ?

File: temba/utils/whatsapp/tests.py (hunk 1 above)
Comment (norkans7) [reply temba/utils/whatsapp/tests.py:280]: So we have the key as empty string?

Comment (rowanseymour) [reply temba/utils/whatsapp/tests.py:280]: we can just omit it

File: temba/utils/management/commands/test_db.py [hunk 3]
```diff
@@ -357,9 +357,21 @@ def create_channels(self, orgs):
...
+                                "params": [{"type": "text"}],
+                            },
+                            "button.1": {
+                                "content": "https://example.com/?wa_organization={{1}}",
+                                "display": "Dish B",
+                                "params": [{"type": "text"}],
+                            },
+                            "button.2": {
+                                "content": "Not Hungry",
+                                "display": "Not Hungry",
```
Comment (rowanseymour) [inline temba/utils/management/commands/test_db.py:372]: remove

**Review by rowanseymour — CHANGES_REQUESTED** [review]

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**